modules/ROOT/partials/configuration/details_initial_state.adoc

```diff
@@ -0,0 +1,51 @@
+[[details_initial_state]]
+== `details_initial_state`
+
+This option sets the state of Accordion sections
+
+* when they are created; and
+* when they are already extant as an editor loads.
+
+When set to `+collapsed+`, all Accordions will present as closed on creation and on the editor loading.
+
+When set to `+expanded+`, all Accordions will present as open on creation and on the editor loading.
+
+When set to `+inherited+`, Accordions preserve their individual state and present as open when created.
+
+The default value for this option is `+inherited+`.
+
+*Type:* `+String+`
+
+*Possible values:* `+inherited+`, `+collapsed+`, `+expanded+`
+
+*Default value:* `+inherited+`
+
+=== Example: using `details_initial_state` to set all created Accordion sections to present as closed by default
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: accordion
+  details_initial_state: 'collapsed'
+});
+----
+
+=== Example: using `details_initial_state` to set pre-configured `<details>` content in new {productname} documents to present as open
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  newdocument_content: '<details><summary>Accordion summary...</summary><p>Accordion body...</p></details>',
```

Comment:
I really think we need to add that `initial_content` option that can set the initial content of the editor. I can see how that would make these types of examples so much easier. :smile: